eiffel-vocabulary/EiffelServiceDeployedEvent.md

```diff
@@ -22,17 +22,17 @@ It implies that the service instance is ready to be started.
 ## Data Members
 ### data.servicename
 __Type:__ String  
-__Required:__ Yes
+__Required:__ Yes  
 __Description:__ The name of the Service that was deployed.
 
 ### data.instancename
```

Comment:
Not really a part of this PR but I noticed that this event does not follow the camelCase syntax. I'll write a separate issue to fix this.